charts/v2.1/cray-hms-rts/templates/jobs.yaml

```diff
@@ -0,0 +1,68 @@
+{{- if .Values.rtsDoInit }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cray-hms-rts-init-deleter
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade,pre-rollback
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+    "helm.sh/hook-weight": "0"
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: “false”
+    spec:
+      restartPolicy: OnFailure
+      serviceAccountName: "{{ .Release.Name | default .Values.rtsDefaultName }}"
+      containers:
+        - name: job-deleter
+          image: "{{ .Values.kubectl.image.repository }}:{{ .Values.kubectl.image.tag }}"
+          imagePullPolicy: "{{ .Values.kubectl.image.pullPolicy }}"
+          command:
+          - /bin/sh
+          - -c
+          - set -x;
+            echo "Deleting Jobs";
+            kubectl -n services delete job cray-hms-rts-init;
+            exit 0
+---
+apiVersion: batch/v1
+kind:       Job
+metadata:
+  name: cray-hms-rts-init
```

Comment:
Should this also have a templated name like the other resoruce?
```
 {{ .Release.Name | default .Values.rtsDefaultName }}-init
```

Which for the JAWS RTS instance that runs the init job I think would end up with the same name `cray-hms-rts-init`

Reply:
The cray-hms-rts-init job needs to have a predictable name so the other deployed RTS charts can all watch the one cray-hms-rts-init job for completion.